configuration/concepts/lab_tests.csv

```diff
@@ -1,24 +1,110 @@
+a24afd60-b6b6-4b7e-82b4-dc599ce62ead,,,Rapid HIV,VIH rápido,VIH rápido,Rapid HIV,,LabTest,Coded,,Positive; Negative;,,,,,,,,,
+dbff115d-32df-4bd5-8265-832e1f3cbe5e,,,Fasting blood glucose,Glucosa en sangre en ayunas,Glucosa en sangre en ayunas,Fasting blood glucose,,LabTest,Numeric,,,,Yes,mg/dL,70,126,,,,
+3a7aefbf-3334-4add-88a0-42ee1c6fd8dd,,,Random blood glucose,Glucosa en sangre aleatoria,Glucosa en sangre aleatoria,Random blood glucose,,LabTest,Numeric,,,,Yes,mg/dL,70,180,,,,
+df5686c1-ad0c-4deb-9919-f77446fd5bb9,,,HbA1C,HbA1C,HbA1C,HbA1C,,LabTest,Numeric,,,,Yes,%,4,6.5,,,,
+1e3e425c-eee2-4d61-8652-dd20239ec159,,,Hemoglobin,Hemoglobina,Hemoglobina,Hemoglobin,,LabTest,Numeric,,,,,,,,,,,
+3f330744-e90c-4a32-9048-07dd255782eb,,,Rapid malaria,Paludismo rápido,Paludismo rápido,Rapid malaria,,LabTest,Coded,,Positive; Negative;,,,,,,,,,
+571e8f4f-4d98-40eb-bd55-c88b8a8ab76d,,,Urine pregnancy test,Prueba de embarazo en orina,Prueba de embarazo en orina,Urine pregnancy test,,LabTest,Coded,,Positive; Negative;,,,,,,,,,
+ab5a7664-ce92-418f-96cb-7a9e0ce9d655,,,EKG,EKG,EKG,EKG,,LabTest,Text,,,,,,,,,,,
+9bb35f8a-f8f0-4256-8b67-7642fb98f3a9,,,Blood culture,Cultura de sangre,Cultura de sangre,Blood culture,,LabTest,Coded,,Positive; Negative,,,,,,,,,
+7f586161-8dff-46e1-9e1d-48424e8510af,,,Stool culture,Cultivo de heces,Cultivo de heces,Stool culture,,LabTest,Coded,,Positive; Negative,,,,,,,,,
+0f2bc510-dd45-414b-a108-5cab86df3b66,,,Wound culture,Cultivo de heridas,Cultivo de heridas,Wound culture,,LabTest,Coded,,Positive; Negative,,,,,,,,,
+f87891da-16d1-4e15-afdf-aa114ab616bf,,,Other culture,Otra cultura,Otra cultura,Other culture,,LabTest,Coded,,Positive; Negative,,,,,,,,,
+334cf7c5-033c-4c0c-b605-7f96c983b51d,,,Fecal occult,Oculto fecal,Oculto fecal,Fecal occult,,LabTest,Coded,,Positive; Negative,,,,,,,,,
+2b2f6511-d428-4c25-86af-f015b495d613,,,Bacteria,Bacterias,Bacterias,Bacteria,,LabTest,Coded,,Positive; Negative,,,,,,,,,
+0a4eafae-296c-4ba6-aef4-a843337aa29b,,,Clue cells,Células de pista,Células de pista,Clue cells,,LabTest,Coded,,Positive; Negative,,,,,,,,,
+d3966267-5882-437b-ae14-18dee9657d14,,,RBCs,Hematíes,Hematíes,RBCs,,LabTest,Coded,,Positive; Negative,,,,,,,,,
+1b985633-13de-4afa-828e-ef38721c660e,,,Ferning,Helecho,Helecho,Ferning,,LabTest,Coded,,Positive; Negative,,,,,,,,,
+dbb2a948-ab68-4474-a24b-5f0ecf697f88,,,Whiff Test,Prueba de olor,Prueba de olor,Whiff Test,,LabTest,Coded,,Positive; Negative,,,,,,,,,
+acdc2f09-2844-43e0-9067-8e71da8319ae,,,Urinalysis,Análisis de orina,Análisis de orina,Urinalysis,,LabSet,N/A,,,PH; Specific gravity; Glucose; Ketones; Nitrites; Leukocyte esterase; Bilirubin; Blood in urine; Protein,,,,,,,,
+4bb7df94-7b92-4e5f-8221-064a76d65e17,,,Yeast,Hongos,Hongos,Yeast,,LabTest,Text,,,,,,,,,,,
+b2a75309-4eb5-414d-9f29-8ecb4ac7dd32,,,Bacterial vaginosis,Vaginosis bacteriana,Vaginosis bacteriana,Bacterial vaginosis,,LabTest,Text,,,,,,,,,,,

```

Comment:
We make this coded with the answers as `Positive;Negative`

Reply:
Done